src/components/modal/ModalAddPatient.tsx

```diff
@@ -17,11 +20,19 @@ const ModalAddPatient: React.FC<ModalAddPatientProps> = ({
   patientState,
   handleChangeNewPatient,
   handleNewPatientSubmit,
+  updatePatientDate,
   onHide,
   show,
   error
 }) => {
   const { t } = useTranslationHook()
+  const birthdayDateValue: Date | null = patientState.birthday !== null && patientState.birthday !== undefined && patientState.birthday !== '' ? new Date(patientState.birthday) : null
```

Comment:
```
const birthdayDateValue: Date | null = patient.birthday ? new Date(patient.birthday) : null;
```

Reply:
done

Reply:
Not really, but agreed to not address to keep the linter happ